README.md

```diff
@@ -456,6 +456,15 @@ To launch the server using the built image, you can run:
 docker run -p 8081:8081 downloadsserver:latest
 ```
 
+## ContentSecurityViolation Detection
+
+The console app includes code which automatically reports CSP violoations to telemetry. This
```

Comment:
```suggestion
CSP violations are automatically reported to telemetry through the console application. This
```
Can we update to say something like this? or
```suggestion
The console application automatically reports CSP violations to telemetry. This
```
I don't think you need to say "includes code" it is kind of implied.